releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,1018 @@
-# Release 1.10
+# Kubernetes 1.10 Release Notes Draft
 
-This will will be used to track release notes for Kubernetes 1.10.
+[[TOC]]
+
+# Major Themes
+
+### Sig-node
+
+Control also figures into changes within SIG-Node, with the beta release of [Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281), the ability to make changes to Kubelet without having to bring down the node, and alpha support for the ability to [configure whether containers in a pod should share a single process namespace](https://github.com/kubernetes/features/issues/495), The CRI has also seen some improvements, with support for  [support for Windows Container Configuration](https://github.com/kubernetes/features/issues/547) and the beta release of the [CRI validation test suite](https://github.com/kubernetes/features/issues/292). In addition, this release includes alpha support for [Debug Containers](https://github.com/kubernetes/features/issues/277).
+
+The Resource Management Working Group graduated three features to Beta in the 1.10 release.  First, [CPU Manager](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/), which allows users to request exclusive CPU cores.  This helps performance in a variety of use-cases, including network latency sensitive applications, as well as applications that benefit from CPU cache residency.  Next, [Huge Pages](https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/), which allows pods to consume either 2Mi or 1Gi Huge Pages.  This benefits applications that consume large amounts of memory.  Use of Huge Pages is a common tuning recommendation for databases and JVMs.  Finally, the [Device Plugin](https://kubernetes.io/docs/concepts/cluster-administration/device-plugins/) feature, which provides a framework for vendors to advertise their resources to the Kubelet without changing Kubernetes core code.  Targeted devices include GPUs, High-performance NICs, FPGAs, InfiniBand, and other similar computing resources that may require vendor specific initialization and setup.
```

Comment:
`Beta` vs `"stable"` vs `stable` used elsewhere

Reply:
changed to beta